lib/module.js

```diff
@@ -2,25 +2,29 @@ const path = require('path')
 const Sentry = require('@sentry/node')
 const WebpackPlugin = require('@sentry/webpack-plugin')
 const deepMerge = require('deepmerge')
+const { Dedupe, ExtraErrorData, RewriteFrames, Transaction } = require('@sentry/integrations')
 
 const logger = require('consola').withScope('nuxt:sentry')
 
 module.exports = function sentry(moduleOptions) {
+  const publicPath = this.options.build.publicPath
+  const buildDirRelative = path.relative(this.options.rootDir, this.options.buildDir)
   const defaults = {
     dsn: process.env.SENTRY_DSN || false,
     disabled: process.env.SENTRY_DISABLED || false,
     disableClientSide: process.env.SENTRY_DISABLE_CLIENT_SIDE || false,
+    disableServerSide: process.env.SENTRY_DISABLE_SERVER_SIDE || false,
     publishRelease: process.env.SENTRY_PUBLISH_RELEASE || false,
     config: {
       environment: this.options.dev ? 'development' : 'production'
     },
     webpackConfig: {
-      include: ['.nuxt/dist/client'],
+      include: [`${buildDirRelative}/dist`],
       ignore: [
         'node_modules',
         '.nuxt/dist/client/img'
       ],
-      urlPrefix: '~/_nuxt/', // TODO: Use publicPath
+      urlPrefix: publicPath.startsWith('/') ? `~${publicPath}` : publicPath,
       configFile: '.sentryclirc'
     }
```

Comment:
While at it, could you also set `release: options.config.release` on the webpack object?
That's because if one configures `release`, then both the one passed to `Sentry.init` and webpack config have to match. Otherwise Sentry will try to guess release name and I don't think it will be able to guess same thing on building and then running in production (as it needs git repo if it's guessing based on commits).

Reference: https://docs.sentry.io/platforms/node/sourcemaps/#uploading-source-maps-to-sentry

Reply:
added